src/main/java/module-info.java

```diff
@@ -1,4 +1,4 @@
-open module net.neoforged.accesstransformer {
+module net.neoforged.accesstransformer {
```

Comment:
Technically this doesn't matter because SJH will make it open anyway. ^^

Reply:
That doesn't mean that other consumers should consider it open.

Reply:
Yeah true